libs/backend/src/models/user/user-profile.ts

```diff
@@ -0,0 +1,28 @@
+import { Schema } from "mongoose";
+
+export const profileSchema = new Schema({
+	picture: {
+		type: String,
+		required: false,
+		trim: true
+	},
+	bio: {
+		type: String,
+		required: false,
+		trim: true
+	},
+	location: {
+		type: String,
+		required: false,
+		trim: true
+	},
```

Comment:
not sure whether or not location is necessary, but guessing it doesn't really matter having it at this point, can be removed later though, if/when it becomes more clear :+1: